src/port1.0/portchecksum.tcl

```diff
@@ -51,14 +51,63 @@ default checksum.skip false
 
 set_ui_prefix
 
+# list_checksum_types
+#
+# Return a list of all checksum types recognized.
+proc portchecksum::list_checksum_types {} {
+    return [list md5 sha1 rmd160 sha256 size]
+}
+
+# list_preferred_checksum_types
+#
+# Return a list of the preferred checksum types recommended for ports.
+proc portchecksum::list_preferred_checksum_types {} {
+    return [list rmd160 sha256 size]
+}
```

Comment:
I figured procs were more robust for exporting, and more versatile in case anyone was going to change just _how_ the list of these algorithms were being determined in `portchecksum` in the future.  Will fix.